actionpack/lib/action_dispatch/http/response.rb

```diff
@@ -96,11 +96,11 @@ def initialize(response, buf)
       end
 
       def body
-        @str_body ||= begin
-                        buf = ''
-                        each { |chunk| buf << chunk }
-                        buf
-                      end
+        begin
```

Comment:
We can't remove the memoization here, this makes the `body` method potentially very expensive, and would break streaming responses.